src/pages/home/home.ts

```diff
@@ -409,6 +414,30 @@ export class HomePage {
       .present();
   }
 
+  leaveGroup() {
+    const alert = this.alertCtrl.create({
```

Comment:
oh shit what's this!?!?! this is awesome!!! we need to change the `confirm` that im using for this instead!